### ISSUE TITLE
Feature ask: do not block client scrolling

### ISSUE THREAD

**Issue by JYC-99** [ankeetmaini/react-infinite-scroll-component#292]
Expected behavior: when user scrolling down, we do not want to block the scrolling. Let the component renders box frames, wait for the async data fetching and backfill those placeholders. 
Is that doable in current version?

**Comment by iamdarshshah** [comment]
Non-blocking scroll with placeholder skeletons requires either virtualization or deferred rendering, both of which are outside the scope of this library. Skeleton/placeholder rendering is best handled in your own `loader` prop while data fetches in the background. Closing as out of scope.
